contracts/HubPool.sol

```diff
@@ -42,48 +49,70 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
     // `slowRelayRoot` to a SpokePool. The latter two roots, once published to the SpokePool, contain
     // leaves that can be executed on the SpokePool to pay relayers or recipients.
     struct RootBundle {
+        // When root bundle challenge period passes and this root bundle becomes executable.
         uint64 requestExpirationTimestamp;
+        // Number of pool rebalance leaves to execute in the `poolRebalanceRoot`. After this number
```

Comment:
this is a bit inconsistent. either we should wrap all variables we refer to in "`"c or do it for none of them. It does not seem like you did this a lot so let's opt for none.
```suggestion
        // Number of pool rebalance leaves to execute in the poolRebalanceRoot. After this number
```